Yoast/wordpress-seo
# 13831 pass presentation instance to presenter filters

## Summary


This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Added the `presentation` instance to presenter filters and updated unit tests accordingly.
* The OG image presenter is a bit different since the image container can possibly contain multiple images. Therefore, a for-loop is needed to pass the presentation instance to the filter for all the images.

I also did some CS things in this PR:
* Added backslashes to PHP-functions in the `src` directory.
* Removed unused helpers from `indexable-presentation`
* Fixed a bug in `og-image-generator`

## Test instructions

This PR can be tested by following these steps:

* To test whether the `presentation` instance is passed, you can take inspiration from [this PR](https://github.com/Yoast/wpseo-video/pull/757).
* Check whether the `og:image` tag is rendered correctly.
* See the unit tests pass.

## UI changes

## Documentation

## Quality assurance


Fixes #13831
